components/bl-radio-buttons-list-component/src/radio-button.js

```diff
@@ -0,0 +1,12 @@
+import { Input } from './input';
+
+export function RadioButton({ isChecked, label, value, handleChange }) {
+  return (
+    <label className="radio-button">
+      <Input isChecked={ isChecked } value={ value } handleChange={ handleChange } />
```

Comment:
isChecked - checked

Usually for properties which occur in browser we use same names:
disabled, checked, required...